hack/check-python/prepare-env.sh

```diff
@@ -9,5 +9,5 @@ function prepare_venv() {
     for req in $(find . -name 'requirements.txt'); do
         python3 "$(which pip3)" install -q -r $req;
     done
-    python3 "$(which pip3)" install -q pydocstyle pyflakes vulture radon
+    python3 "$(which pip3)" install -q pydocstyle pyflakes vulture radon flake8_polyfill
```

Comment:
In order to avoid similar surprise in the future, we should pin the dependency versions. I would propose that we extract the dependencies into `requirement.txt` and pin the versions. We should apply the same to `test/acceptance/features/requirements.txt` as well.